### PR TITLE
Set up cucumber

### PR DESCRIPTION
**Pull request by velimd**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CTF-10

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/e2e/steps/homepage.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Given, Then } from 'cucumber';
+import { expect } from 'chai';
+
+Given('I am on FACT homepage', function() {
+  return this.driver.get('http://localhost:3100/');
```
Comment (linusnorton) [inline src/test/e2e/steps/homepage.ts:5]: Make this configurable to TEST_URL env var for the pipeline

File: src/test/config.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const config = {
```
Comment (nigeldunne) [inline src/test/config.ts:1]: There is a already a config file in config/test.json

File: package.json [hunk 3]
```diff
@@ -15,6 +15,7 @@
     "test:coverage": "jest --coverage",
     "test:routes": "jest -c jest.routes.config.js",
     "test:a11y": "jest -c jest.a11y.config.js --maxWorkers 15",
+    "test:e2e": "cucumber-js",
```
Comment (nigeldunne) [inline package.json:18]: Are these tests intended to replace the functional tests?

File: src/test/e2e/features/homepage.feature [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Feature: Homepage
+
+  Scenario: Load Homepage
+    Given I am on FACT homepage
+    Then I expect the page header to be "Find a Court or Tribunal"
```
Comment (nigeldunne) [inline src/test/e2e/features/homepage.feature:5]: Page title is now "Find a court or tribunal - GOV.UK"

File: package.json (hunk 3 above)
Comment (nigeldunne) [reply package.json:18]: The jenkins pipeline calls test:functional

Comment (velimd) [reply package.json:18]: hmm, tbh i assumed so as in probate, functional test would call e2e however, i dont think theyre the same.

File: src/test/config.ts (hunk 2 above)
Comment (velimd) [reply src/test/config.ts:1]: not sure if we were gonna keep it as all the configs ive seen in hmcts repos are all yaml files and this typescript file allows me to check the environment the service is in.

**Review by nigeldunne — APPROVED** [review]

**Pull request closed without merging**